.storybook/manager.ts

```diff
@@ -1,8 +1,8 @@
-import { addons, types } from "@storybook/manager-api";
+import { addons, types } from "storybook/manager-api";
 import sageTheme from "./sage-docs-theme";
 import { ADDON_ID, TOOL_ID } from "./version-picker/constants";
 import { VersionPicker } from "./version-picker";
-import { API_PreparedIndexEntry, API_StatusObject } from "@storybook/types";
+import { API_PreparedIndexEntry } from "storybook/internal/types";
```

Comment:
Question: is this still needed? If it is, is it the appropriate type to use given that it's now `internal`?